app/controllers/idv/agreement_controller.rb

```diff
@@ -41,6 +41,7 @@ def analytics_arguments
       {
         step: 'agreement',
         analytics_id: 'Doc Auth',
+        skip_hybrid_handoff: idv_session.skip_hybrid_handoff,
```

Comment:
Is this duplicative since AbTestAnalyticsConcern does the same thing?

Reply:
My thought was that we want skip_hybrid_handoff everywhere for the GettingStarted A/B test, but once we pull that out, it still makes sense to log it where it affects the code flow, so I put those in separately.